website/integrations/services/jellyfin/index.md

```diff
@@ -87,6 +89,107 @@ At this point, enter a username and click "Save Search Attribute Settings and Qu
 1. Click "Save"
 2. Logout, and login with a LDAP user. Username **must** be used, logging in with email will not work.
 
-## authentik Configuration
+### Authentik Configuration
 
 No additional authentik configuration needs to be configured. Follow the LDAP outpost instructions to create an LDAP outpost and configure access via the outpost
+
+## OIDC Configuration
+
+### Authentik Configuration
+
+To use the role claim within Jellyfin we will have to add a custom property. This is not needed but does allow admins to be set for SSO users automatically. It can also deny access if they don't have a role, but this should be done though Authentik instead.
```

Comment:
```suggestion
To use the role claim within Jellyfin we will have to add a custom property. This is not needed but does allow admins to be set for SSO users automatically. It can also deny access if they don't have a role, but this should be done though authentik instead.
```